src/components/Progress.js

```diff
@@ -1,12 +1,22 @@
 import React from 'react';
+import { CircularProgressbar } from 'react-circular-progressbar';
+import 'react-circular-progressbar/dist/styles.css';
+import '../assets/styles/progress.css';
 
-const Progress = (prop) => {
-  const { book } = prop;
+const Progress = () => {
+  // const { book } = prop;
```

Comment:
- [ ] <b>REQUIRED:</b> To make sure that your codebase is always clean, remember to remove commented-codes.